content/responses/file_request.yml

```diff
@@ -0,0 +1,157 @@
+
+      This defaults to title of the file request that was
+      copied to create this file request.
+    example: "Please upload documents"
+
+  description:
+    type: string
+    nullable: true
+    description: |-
+      The optional description of this file request. This is
+      shown in the Box UI to users uploading files.
+
+      This defaults to description of the file request that was
+      copied to create this file request.
+    example: "Following documents are requested for your process"
+
+  status:
+    type: string
+    example: active
+    description: |-
+      The status of the file request. This defaults
+      to `active`.
+
+      When the status is set to `inactive`, the file request
+      will no longer accept new submissions, and any visitor
+      to the file request URL will receive a `HTTP 404` status
+      code.
+
+      This defaults to status of file request that was
+      copied to create this file request.
+    enum:
+      - active
+      - inactive
+
+  is_email_required:
+    type: boolean
+    example: true
+    description: |-
+      Whether a file request submitter is required to provide
+      their email address.
+
+      When this setting is set to true, the Box UI will show
+      an email field on the file request form.
+
+      This defaults to setting of file request that was
+      copied to create this file request.
+
+  is_description_required:
+    type: boolean
+    example: true
+    description: |-
+      Whether a file request submitter is required to provide
+      a description of the files they are submitting.
+
+      When this setting is set to true, the Box UI will show
+      a description field on the file request form.
+
+      This defaults to setting of file request that was
+      copied to create this file request.
+
+  expires_at:
+    type: string
+    format: date-time
+    description: |-
+      The date after which a file request will no longer accept new
+      submissions.
+
+      After this date, the `status` will automatically be set to
+      `inactive`.
+    example: "2020-09-28T10:53:43-08:00"
+
+  folder:
+    allOf:
+      - $ref: '#/components/schemas/Folder--Mini'
+      - description: |-
+          The folder that this file request is associated with.
+
+          Files submitted through the file request form will be
+          uploaded to this folder.
+    nullable: false
+
+  url:
+    type: string
+    description: |-
+      The generated URL for this file request. This URL can be shared
+      with users to let them upload files to the associated folder.
+    example: https://cloud.app.box.com/f/19e57f40ace247278a8e3d336678c64a
+    readOnly: true
+
+  etag:
+    type: string
+    example: "1"
+    nullable: true
+    description: |-
+      The HTTP `etag` of this file. This can be used within some API
+      endpoints in the `If-Match` and `If-None-Match` headers to only
+      perform changes on the file request if (no) changes have happened.
```

Comment:
@virenderg do any of the new APIs support the `If-Match`/`If-None-Match` headers? If so, I can add them to those endpoints.

Reply:
@cbetta , Yes the PUT endpoint. I added If-Match and 412 error codes in previous commit (https://github.com/box/box-openapi/blob/1040e562a665a94954b942c219836bb491604d28/content/paths/file_requests__put_file_requests_id.yml#L19). Let me know if other changes are needed.

Reply:
Ah you already did. I simplified the language here a bit so it is a bit more specific to this usecase, and only mentions the `If-Match` header (cause I doubt the `If-None-Match` header makes sense here).